Model/PostalCodeType.cs

```diff
@@ -0,0 +1,9 @@
+namespace CustomsController.Model
+{
+    public enum PostalCodeType
```

Comment:
summary

Reply:
"Exception" is a pretty heavily used and well-defined word in programming, can we use a synonym that isn't a programming keyword?

Reply:
SpecialCase